packages/esm-ward-app/src/ward-view/ward-view.resource.ts

```diff
@@ -30,19 +30,20 @@ export function filterBeds(admissionLocation: AdmissionLocationFetchResponse): B
   const bedLayouts = admissionLocation.bedLayouts
     .filter((bl) => bl.bedId)
     .sort((bedA, bedB) => collator.compare(bedA.bedNumber, bedB.bedNumber));
+
   return bedLayouts;
 }
 
 //TODO: This implementation will change when the api is ready
-export function getWardMetrics(beds: Bed[], wardPatientGroup: WardPatientGroupDetails): WardMetrics {
+export function getWardMetrics(bedLayouts: BedLayout[], wardPatientGroup: WardPatientGroupDetails): WardMetrics {
   const bedMetrics = {
     patients: '--',
     freeBeds: '--',
     capacity: '--',
   };
-  if (beds == null || beds.length == 0) return bedMetrics;
-  const total = beds.length;
-  const occupiedBeds = beds.filter((bed) => bed.status === 'OCCUPIED');
+  if (bedLayouts == null || bedLayouts.length == 0) return bedMetrics;
+  const total = bedLayouts.length;
+  const occupiedBeds = bedLayouts.filter((bed) => bed.patients.length);
```

Comment:
Maybe a more explicit boolean check?

```suggestion
  const occupiedBeds = bedLayouts.filter((bed) => bed.patients.length > 0);
```